image/tools/lib/encryption/gpg.sh

```diff
@@ -1,6 +1,6 @@
 function check_encryption_enabled {
     local result=$(oc get secret -n ${ENCRYPTION_SECRET_NAMESPACE} ${ENCRYPTION_SECRET_NAME} -o template --template='{{.metadata.name}}')
-    if [[ "$result" == "${ENCRYPTION_SECRET_NAME}" ]]; then
+    if [ "$result" == "${ENCRYPTION_SECRET_NAME}" ]; then
```

Comment:
You used `-eq` in other conditions, why not here?

Reply:
whoops forgot to change it